cognite/neat/_rules/transformers/_converters.py

```diff
@@ -447,6 +454,28 @@ def _to_enterprise(self, reference_model: DMSRules) -> JustRules[DMSRules]:
 
         return JustRules(enterprise_model)
 
+    @staticmethod
+    def _expand_properties(rules: DMSRules) -> DMSRules:
+        probe = DMSAnalysis(rules)
+        ancestor_properties_by_view = probe.classes_with_properties(
+            consider_inheritance=True, allow_different_namespace=True
+        )
+        property_ids_by_view = {
+            view: {prop.view_property for prop in properties}
+            for view, properties in probe.classes_with_properties(consider_inheritance=False).items()
+        }
+        for view, property_ids in property_ids_by_view.items():
+            ancestor_properties = ancestor_properties_by_view.get(view, [])
```

Comment:
I guess you call this method once you pull the referenced views ? as otherwise rules will not contain ancestor properties which are in different space